Brewfile

```diff
@@ -97,3 +97,4 @@ brew "wdiff"
 brew "wget"
 brew "z"
 brew "zsh"
+brew "node"
```

Comment:
**issue:** Homebrew is not actually recommended as a way to install Node. It can be the reason for strange behavior.